tests/competency_questions/physical/soundness/001_biofuel_is_renewable_fuel.omn

```diff
@@ -3,11 +3,12 @@ Prefix: : <http://openenergy-platform.org/ontology/oeo/>
 Ontology: <http://openenergy-platform.org/ontology/cc/> <http://openenergy-platform.org/ontology/cc/>
 
 # Competency Question: Is  biofuel renewable fuel?
- 
+
 Class: OEO_00000033
 Class: OEO_00000072
 
 Class: OEO_00000072
   SubClassOf: OEO_00000033
-    
-# This competency question gets deprecated with this pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1409
\ No newline at end of file
+
+# This competency question gets deprecated with this pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1409
+# This means it should be false
```

Comment:
To me, the answer this competency question is neither false not right, but the question is undecidable, because one needs additional information about the biofuel to decide this question. Same with Q29.

Reply:
I think that is a good solution.